src/app.rb

```diff
@@ -0,0 +1,109 @@
+require_relative 'book'
+require_relative 'rental'
+require_relative 'teacher'
+require_relative 'student'
+
+def list_options
+  puts 'Welcome to School library App!'
+  puts "
+    Please choose an option by entering a number:
+    1- List all books
+    2- List all patron
+    3- Create a person
+    4- Create a book
+    5- Create a rental
+    6- List all rentals for a given person id
+    7- Exit"
+end
+
+def options(books, patron, rentals)
+  loop do
+    list_options
+    option = gets.chomp.to_i
+    case option
+    when 1
+      books_list(books)
+    when 2
+      patrons_list(patron)
+    when 3
+      create_person(patron)
+    when 4
+      create_book(books)
+    when 5
+      create_rental(books, patron, rentals)
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/164734847-25e80623-65f1-4b27-aa3a-3639e043457b.png)


- [ ] Kindly make sure your parameters are not reversed, your app is not able to create a rental because you reversed the parameters of the `create_rental` method. 

your `create_rental` method 
```rb
def create_rental(books, rentals, patron)
# your logic.......
end
```
See now how you are calling it here 
```.rb
...
   when 5
      create_rental(books, patron, rentals)
...
```
instead of 
```.rb
...
    when 5
      create_rental(books, rentals, patron)
...
```

Reply:
@andersonlebon   Thank you for showing me this.